gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-account/src/main/java/io/gravitee/am/gateway/handler/account/resources/account/AccountEndpointHandler.java

```diff
@@ -122,6 +153,153 @@ public void updateProfile(RoutingContext routingContext) {
         }
     }
 
+    public void enrollFactor(RoutingContext routingContext) {
```

Comment:
When I try to enroll a factor (using OTP), the Gateway never respond.

Reply:
I'll retest, and probe for insight and come back to it in the am meeting if I can't figure it out